cmd/exec.go

```diff
@@ -239,7 +226,7 @@ func execRun(cmd *cobra.Command, args []string) error {
 
 	// Forward SIGINT, SIGTERM, SIGKILL to the child command
```

Comment:
Please update this comment to match https://github.com/segmentio/aws-okta/pull/243/files#r346352432